articles/azure-resource-manager/troubleshooting/error-storage-account-name.md

```diff
@@ -129,4 +130,13 @@ name: '${storageNamePrefix}${uniqueString(resourceGroup().id)}'
 "name": "[concat(parameters('storageNamePrefix'), uniquestring(resourceGroup().id))]"
 ```
 
+## Cause 2
+
+The storage account was recently deleted.
+   - If a request to create the storage account comes from a different subscription and tenant than where it was previously located, it will be denied for security purposes as described here, [Prevent dangling DNS entries and avoid subdomain takeover](https://docs.microsoft.com/en-us/azure/security/fundamentals/subdomain-takeover). 
```

Comment:
```suggestion
   - If a request to create the storage account comes from a different subscription and tenant than where it was previously located, it will be denied for security purposes as described here, [Prevent dangling DNS entries and avoid subdomain takeover](https://docs.microsoft.com/azure/security/fundamentals/subdomain-takeover). 
```
remove en-us to resolve the warning